tests/simple_tests.rs

```diff
@@ -293,4 +334,23 @@ fn test_signed_comparison_8bit_cmp(){
         hlt");
     assert_eq!(vm.reg8(Reg8::AL), 0xFE);
     assert_eq!(vm.flags, X86Flags{carry: true, sign: true, adjust: true, parity: true, ..Default::default()});
+}
+
+#[test]
+fn test_inc_and_dec_8bit_and_32bit() {
```

Comment:
Can you also add a test ensuring that inc/dec does not modify the carry flag?

Reply:
sure.